django-stubs/contrib/admindocs/views.pyi

```diff
@@ -17,5 +18,7 @@ class TemplateDetailView(BaseAdminDocsView): ...
 
 def get_return_data_type(func_name: Any): ...
 def get_readable_field_data_type(field: Union[Field, str]) -> str: ...
-def extract_views_from_urlpatterns(urlpatterns: Any, base: str = ..., namespace: Optional[Any] = ...): ...
+def extract_views_from_urlpatterns(
+    urlpatterns: Sequence[Union[URLPattern, URLResolver]], base: str = ..., namespace: Optional[str] = ...
```

Comment:
Looks like `Iterable` to me: https://github.com/django/django/blob/7119f40c9881666b6f9b5cf7df09ee1d21cc8344/django/contrib/admindocs/views.py#L462